core/lib/zksync_core/src/db_pruner/mod.rs

```diff
@@ -0,0 +1,499 @@
+mod metrics;
+pub mod prune_conditions;
+
+use std::{fmt, sync::Arc, time::Duration};
+
+use async_trait::async_trait;
+use tokio::sync::watch;
+use zksync_dal::ConnectionPool;
+use zksync_types::L1BatchNumber;
+
+use crate::db_pruner::metrics::{MetricPruneType, METRICS};
+
+#[derive(Debug)]
+pub struct DbPrunerConfig {
+    pub soft_and_hard_pruning_time_delta: Duration,
+    pub next_iterations_delay: Duration,
+    pub pruned_chunk_size: u32,
```

Comment:
Bikeshedding: I'd call this `pruned_batch_chunk_size`, so that it's clear what unit is used for measuring this size.

Reply:
corrected